addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6223,6 +6223,13 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
      */
     onBuilt() {
         this._patchShape(this.$target[0]);
+        // Flip classes should no longer be used but are still present in some
+        // theme snippets.
+        if (this.$target[0].querySelector('.o_we_flip_x, .o_we_flip_y')) {
+            this._handlePreviewState(false, () => {
+                return {flip: this._getShapeData().flip};
+            });
+        }
```

Comment:
There should be two commits about this fix, no? :)

Reply:
@qsm-odoo A second commit for this change you mean ?